frontend/public/components/autocomplete.tsx

```diff
@@ -8,9 +8,9 @@ import { TextFilter } from './factory';
 
 const MAX_SUGGESTIONS = 5;
 
-const labelParser = (resources: K8sResourceCommon[]): Set<string> => {
+const labelParser = (resources: K8sResourceCommon[], labelPath: string): Set<string> => {
```

Comment:
```suggestion
const labelParser = (resources: any, labelPath: string): Set<string> => {
```
Since monitoring is strictly not passing K8sResource.